### ISSUE TITLE
ERROR: Could not find a version that satisfies the requirement tensorflow-gpu

### ISSUE THREAD

**Issue by JamesLogan2KB** [CorentinJ/Real-Time-Voice-Cloning#325]
When I type:
`pip install -r requirements.txt`

I get reror:
```
ERROR: Could not find a version that satisfies the requirement tensorflow-gpu<=1.14.0,>=1.10.0 (from -r requirements.txt (line 1)) (from versions: none)
ERROR: No matching distribution found for tensorflow-gpu<=1.14.0,>=1.10.0 (from -r requirements.txt (line 1))
```
How to fix this?

MacOS Catalina.

**Comment by volkb** [comment]
I am also getting this error

Running Windows 10 and Python 3.8

**Comment by sjmelsom** [comment]
I don't think that tensorflow 1.14 is available from pypl anymore. You will need to download a release of 1.14 directly from the tensorflow github repo.

**Comment by denisuk227** [comment]
Same problem. I install python 3.7.7 (64bit) and it start download.
But there are a lot of other problems, i spent 2 days to done it, but it didn't work.
I install MS VS Community 2017, Cuda 10, cudnn7, and other solutions from stackoverflow and it don't start working(((

**Comment by maxwyb** [comment]
For those on Windows 10, using Python 3.7 + CUDA 10.0 + their corresponding versions of cuDNN and PyTorch have worked for me. I followed [this installation guide](https://poorlydocumented.com/2019/11/installing-corentinjs-real-time-voice-cloning-project-on-windows-10-from-scratch/) step by step.

I tried that TensorFlow 1.14 would not work with the newest CUDA 10.2 - a DLL error occurs when importing the library.

**Comment by Haven-Net** [comment]
This woked for me 
conda install -c conda-forge tensorflow-gpu

**Comment by blue-fish** [comment]
Tensorflow-gpu is [no longer supported on mac](https://stackoverflow.com/q/44744737) so `pip install -r requirements.txt` will fail on tensorflow-gpu==1.15

However, tensorflow-gpu is no longer a hard requirement since we added CPU support. I'll submit a pull request to resolve this issue.

**Comment by blue-fish** [comment]
Resolved by #399 .
